Sources/PubNub/Helpers/Crypto/Miscellaneous/CryptoInputStream.swift

```diff
@@ -26,11 +26,30 @@
 //
 
 import Foundation
+import CommonCrypto
 
 /// A stream that provides read-only stream functionality while performing crypto operations
 public class CryptoInputStream: InputStream {
   // swiftlint:disable:previous type_body_length
-
+  public struct DataSource {
```

Comment:
So, right now CrytoInputStream will work only for our cryptors? In future, users also would like to have some help from use in handling stream chunks reading.

Reply:
I still preserved this class as `public` so customers can use it. What's changed is the input that this class takes. I decoupled "old" `Crypto` from this class so now customers may create its instances without any dependencies to struct/classes we're going to deprecate.